legend-engine-xt-documentStore-pure/src/main/resources/core_documentstore/graphFetch/documentStoreGraphFetch.pure

```diff
@@ -123,18 +134,18 @@ function meta::external::store::document::graphFetch::executionPlan::planRootGra
       batchSize      = $batchSize,
       //tempTableName  = tempTableName(0),
       //columns        = $sqlNode.resultColumns->filter(x | $x.label->startsWith('"pk_'))->map(x | ^SQLResultColumn(label = $x.label->substring(1, $x.label->length() - 1), dataType = $x.dataType->toOne())),
-      children       = $rootTree->generateRelationalChildGraphNodes($rootTree->nodePathName(), $setImpls, $simplePrimitiveProperties, $orderedPaths, $store, $mapping, $oneRuntime, $sq.inScopeVars, $exeCtx, $extensions, $debug),
-      //executionNodes = $wrappedNode,
+      //children       = $rootTree->generateRelationalChildGraphNodes($rootTree->nodePathName(), $setImpls, $simplePrimitiveProperties, $orderedPaths, $store, $mapping, $oneRuntime, $sq.inScopeVars, $exeCtx, $extensions, $debug),
```

Comment:
Assuming this is only when we have "true" joins??

Reply:
yes I removed children because it routes to the function (which kinda works now since I worked on it) that build the queries for non primitive properties (that is joins mainly I believe)